website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,74 @@
+### Permissions 
+
+When a user does not have CRUD (Create, Read, Update and Delete) permissions, hide the related actions.
+
+In this example, the user's role is **viewer** and they cannot **deploy** or **destroy** a cluster. An explanation is not needed as users understand that their account has limited permissions and that they require an additional user's help to gain these permissions (often someone with admin or similar roles).
+
+![Image showing two dropdown with lists open, one with only one menu item that is reader specific, while the other has more options that allow for further capabilities.](/assets/patterns/disabled-patterns/permissions-example.png)
+
+In scenarios where a user lands on a page they cannot access, provide a reason why with guidance so that users understand what steps they can take to rectify the error. So that users don't feel stuck, provide them a call to action to facilitate a path forward.
```

Comment:
[suggestion] This adds a bit of brevity I think, but I'm not overly opinionated about it.

```suggestion
In scenarios where a user lands on a page they cannot access, provide a reason why with guidance and steps to rectify the error. So that users don't feel stuck, provide them a call to action to facilitate a path forward.
```